examples/example_create_zarr_nwb.py

```diff
@@ -0,0 +1,121 @@
+from typing import Any
+import shutil
+import os
+import zarr
+import pynwb
+import lindi
+
+
+def example_create_zarr_nwb():
+    zarr_dirname = 'example_nwb.zarr'
+    if os.path.exists(zarr_dirname):
+        shutil.rmtree(zarr_dirname)
+
+    nwbfile = _create_sample_nwb_file()
+
+    store = zarr.DirectoryStore(zarr_dirname)
+    zarr.group(store=store)  # create a root group
+    with lindi.LindiH5pyFile.from_zarr_store(store, mode='a') as client:
```

Comment:
```suggestion
    with lindi.LindiH5pyFile.from_zarr_store(store, mode='r+') as client:
```

`"a" mode is not implemented below yet

Reply:
fixed